src/syngen/ml/vae/models/dataset.py

```diff
@@ -864,7 +913,7 @@ def _preprocess_nan_cols(
                 fillna_value = self.df[feature].mean()
             elif fillna_strategy == "mode":
                 fillna_value = self.df[feature].dropna().mode().sample(1).values[0]
-            elif fillna_strategy == "text":
+            elif (fillna_strategy == "text") or (fillna_strategy == "email"):
```

Comment:
I think that it's not necessary to add new fill_strategy - "email"

Reply:
fillna_strategy takes part in formulating of log records (line 910), so it should be separate for email.